lib/util.js

```diff
@@ -265,25 +265,28 @@ function fixTemplateLiteral(node, lines) {
     // If there are no quasi elements, then there is nothing to fix.
     return;
   }
+  
+  // node.loc is not present when using export default with a template literal
```

Comment:
Any idea why `node.loc` would be missing?

Reply:
Not sure to be honest. I can reproduce this with the regression test however. I also can reproduce this with `@babel/parser`, even with `ranges` turned on.

Reply:
Ahh, that's something we do in a few places to disable reprinting problematic nodes, e.g. [here](https://github.com/benjamn/recast/blob/45773b02ad1dfbd61ac0050df17587d0d83dca7e/lib/util.js#L186-L188).

This PR is definitely safe/fine for now. Sorry for the wait!